src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -426,22 +432,14 @@ private void TrackTaintedDataEnteringSink(SymbolAccess sink, IEnumerable<SinkKin
             /// Determines if tainted data is entering a sink as a method call or constructor argument, and if so, flags it.
             /// </summary>
             /// <param name="targetMethod">Method being invoked.</param>
-            /// <param name="taintedArguments">Arguments with tainted data to the method.</param>
+            /// <param name="visitedArguments">Arguments to the method.</param>
             /// <param name="originalOperation">Original IOperation for the method/constructor invocation.</param>
             private void ProcessTaintedDataEnteringInvocationOrCreation(
```

Comment:
>ProcessTaintedDataEnteringInvocationOrCreation [](start = 25, length = 46)

Probably should come up with a better name (and <summary>) since this isn't looking at only tainted arguments anymore.